src/stp/plays/defensive/DefendShot.cpp

```diff
@@ -23,52 +23,118 @@ DefendShot::DefendShot() : Play() {
 
     roles = std::array<std::unique_ptr<Role>, stp::control_constants::MAX_ROBOT_COUNT>{
         std::make_unique<role::Keeper>(role::Keeper("keeper")),
-        std::make_unique<role::Defender>(role::Defender("defender_1")),
-        std::make_unique<role::Defender>(role::Defender("defender_2")),
-        std::make_unique<role::Harasser>(role::Harasser("harassing_defender")),
+        std::make_unique<role::Defender>(role::Defender("waller_1")),
```

Comment:
The role names seem a bit confusing. Wallers and midfielders have the Defender role, whilst defenders (and the harasser) have the Harasser role. Do you think the naming could be improved here? Or are the names of the roles them selves not descriptive enough?